build.rs

```diff
@@ -0,0 +1,14 @@
+use cfgrammar::yacc::YaccKind;
+use lrlex::CTLexerBuilder;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I did not know that main() could have that signature! Today I learned.

Reply:
stolen from https://softdevteam.github.io/grmtools/master/book/quickstart.html